### DISCUSSION TITLE
Welcome to dmrtc Discussions!

### DISCUSSION THREAD

**Discussion by kantooon** [qradiolink/dmrtc#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alwinhb** [comment]
@kantooon Hi, Can you help me with some DMR hex samples to decode 

1. Actual data samples (264 bits) that contains some "hi" or test message.
2. Voice samples from A to F with LC header and terminator

**Reply by kantooon** [reply]
Sorry, I can't provide hex captures  I have no means to do so. All I can give is recordings of a control channel which can be played back using dsd-fme.

**Reply by alwinhb** [reply]
Sure that would work! Can you share those as attachments @kantooon
